packages/components/organisms/f-header/src/components/Logo.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-    <a
+    <component
+        :is="wrapperComponent"
         :aria-label="linkAltText"
         href="/"
```

Comment:
This would render out for the span. You may want to have a condition here too.

Reply:
This goes for `aria-label` and `data-trak` attributes too. You could get all for one by doing something like:
```js
    v-bind="linkProperties"
    // ...
    computed: {
        // ...
        linkProperties() {
            return this.isLogoDisabled ? {} : {
                'aria-label': this.linkAltText,
                href: '/',
                'data-trak': `{
                    "trakEvent": "click",
                    "category": "engagement",
                    "action": "header",
                    "label": "${logoGtmLabel}"
                }`
            }
        }
    }
```

Reply:
Done 🙂